### PR TITLE
Styling fixes within filters, hierarchy fix on organisation members b…

### PR DESCRIPTION
**Pull request by HeleenSG**
### Changes
- Styling fixes within filters (Quick fixes as we work towards new visual implementation of filters)
- Hierarchy fix on organisation members button. Changed a primary button into a secundair button as it isn't the primary function/call to action on the page. 
- Fixes inconsistent text colors within forms. From black to the dark grey as specified in the designs. 
- Adds columns within clearance level filters (ooi page)

Todo: 
Add filter counter to: resolved in #3355
- [x] Members page
- [x] Object page
- [x] Findings page
- [x] KAT-alogus
- [x] Reports
- [x] OOI tree
- [x] OOI graph
- [x] Tasks

### Issue link
Closes https://github.com/minvws/nl-kat-coordination/issues/3274

### Demo
Updated filter styling: 
Screenshot 2024-08-06 at 11 42 53

Screenshot 2024-08-06 at 11 53 11

secundary button "Add member(s)"
Before: 
Screenshot 2024-08-06 at 11 42 41

After:
Screenshot 2024-08-06 at 11 46 24

Columns on clearance level filters object list: 
Screenshot 2024-08-06 at 12 25 15

### QA notes

- Check if implementation is according to design: https://www.figma.com/design/DELH5exdYs61olBbZFpk34/%E2%9C%85-KAT-%7C-Approved-%7C-Patterns?node-id=2302-2284&m=dev

---

### Code Checklist

- [ ] All the commits in this PR are properly PGP-signed and verified.
- [ ] This PR only contains functionality relevant to the issue.
- [ ] I have written unit tests for the changes or fixes I made.
- [ ] I have checked the documentation and made changes where necessary.
- [ ] I have performed a self-review of my code and refactored it to the best of my abilities.

- [ ] Tickets have been created for newly discovered issues.
- [ ] For any non-trivial functionality, I have added integration and/or end-to-end tests.
- [ ] I have informed others of any required `.env` changes files if required and changed the `.env-dist` accordingly.
- [ ] I have included comments in the code to elaborate on what is not self-evident from the code itself, including references to issues and discussions online, or implicit behavior of an interface.

---

## Checklist for code reviewers:

Copy-paste the checklist from [the docs/source/templates folder](https://github.com/minvws/nl-kat-coordination/blob/main/docs/source/templates/pull_request_template_review_code.md) into your comment.

---

## Checklist for QA:

Copy-paste the checklist from [the docs/source/templates folder](https://github.com/minvws/nl-kat-coordination/blob/main/docs/source/templates/pull_request_template_review_qa.md) into your comment.

### CODE REVIEW

File: rocky/rocky/views/scheduler.py [hunk 1]
```diff
@@ -49,10 +49,14 @@ def get_form_data(self) -> dict[str, Any]:
...
         return {k: v for k, v in form_data.items() if v}
 
+    def count_active_filters(self):
+        return len(self.get_form_data())
+
     def get_task_filter_form(self) -> TaskFilterForm:
         return self.task_filter_form(self.request.GET)
 
     def get_task_list(self) -> LazyTaskList | list[Any]:
+        self.count_active_filters()
```
Comment (ammar92) [inline rocky/rocky/views/scheduler.py:59]: This statement has no effect
```suggestion
```

**Review by NoemiOverdijk — APPROVED** [review]

File: rocky/rocky/views/scheduler.py (hunk 1 above)
Comment (Rieven) [reply rocky/rocky/views/scheduler.py]: Forgot to remove this

File: rocky/tests/test_crisis_room.py [hunk 2]
```diff
@@ -31,13 +29,16 @@ def test_crisis_room_observed_at(rf, client_member, mock_crisis_room_octopoes):
     request.resolver_match = resolve(reverse("crisis_room"))
     response = CrisisRoomView.as_view()(request)
     assert response.status_code == 200
-    assertContains(response, "Jan 01, 2021")
```
Comment (Rieven) [inline rocky/tests/test_crisis_room.py:34]: Removed this date before filter
image

Comment (Rieven) [reply rocky/tests/test_crisis_room.py:34]: Moved to top:
image

File: rocky/rocky/views/scheduler.py [hunk 3]
```diff
@@ -34,13 +34,21 @@ def get_task_filters(self) -> dict[str, Any]:
...
 
-    def get_form_data(self) -> dict[str, Any]:
+    def get_task_filter_form_data(self) -> dict[str, Any]:
         form_data = self.get_task_filter_form().data.dict()
         return {k: v for k, v in form_data.items() if v}
 
+    def count_active_task_filters(self):
+        form_data = self.get_task_filter_form_data()
+
+        for task_filter in form_data.copy():
```
Comment (ammar92) [inline rocky/rocky/views/scheduler.py:47]: You don't need to copy an object to modify it and calculate the sum of its elements. Instead, you could count the original elements and subtract the filtered elements. Copying is usually expensive and should be avoid if not necessary. For example:

```python3
count = len(form_data)
for task_filter in form_data:
    if task_filter in ("observed_at", "ooi_id"):
        count -= 1

return count
```

Or functional:

```python3
return sum((task_filter for task_filter in form_data if task_filter not in ("observed_at", "ooi_id")))
```

**Review by ammar92 — COMMENTED** [review]
One tiny remark, looks good otherwise

**Review by underdarknl — APPROVED** [review]

**Pull request merged**